src/plugins/intel_npu/tools/protopipe/src/parser/config.cpp

```diff
@@ -302,6 +346,12 @@ struct convert<GlobalOptions> {
 template <>
 struct convert<OpenVINOParams> {
     static bool decode(const Node& node, OpenVINOParams& params) {
+        std::cout << "Septi: " << __FILE__ << ":" << __LINE__ << "   ---   " << __func__ << std::endl;
+
+        const std::unordered_set<std::string> parameters = {
+            "config", "connections", "device", "il", "iml", "ip", "name", "nireq", "ol", 
+            "oml", "op", "op_desc", "path", "priority", "repeat_count", "reshape", "tag", "type"};
```

Comment:
`exec_time_in_us` isn't omitted there?

Reply:
Nothing done.
String: "exec_time_in_us" was not found. Is there a typo?